### DISCUSSION TITLE
How to Play a Montage in Overlay Mode?

### DISCUSSION THREAD

**Discussion by scnon** [Sixze/ALS-Refactored#517] [Q&A]
I'm attempting to play a montage using the blueprint version, but it's not working. Does anyone have any insights or solutions?

**Comment by GStreetGames** [comment]
Had this problem a while ago. This solved it:
https://youtu.be/7asj3nJAZxc?si=rpAA0JBZ_qWIBRSg
Hope that helps!

**Reply by scnon** [reply]
Thank you for your response.
But I found that in the Blueprint version, the OverriderLayer slot can be used. However, in the Refactored version, the Layer slot cannot play, and only the Grounded and Locomotion slot groups can properly play animation montages. Therefore, I cannot use the Layer slot to control playing only the upper body animations.
I tried comparing the differences between the animation blueprints of the two versions, and I found that apart from isolating all the OverlayState state machines into a separate ALSLinkedAnimInstance, there doesn't seem to be much change. Therefore, I don't understand why the montage cannot play properly when set up according to the Blueprint version.

**Reply by GStreetGames** [reply]
Ah, sorry about that, I thought you were exclusively working with the blueprint version. I believe that this version utilizes the Gameplay Ability System, but hopefully someone more knowledgeable of this version can clarify. Sorry I couldn't be of more help, I'm still learning this system myself.

**Reply by scnon** [reply]
It's fine. I am just looking for some tutorials on ALS, but most of what I found are about the blueprint version.

**Comment by Sheardo** [comment]
I'm also facing this issue and haven't figured it out

**Comment by jomath229** [comment]
If you want to play a montage using ALS refactored over an overlay, this has already been discussed here:

https://github.com/Sixze/ALS-Refactored/discussions/196#discussioncomment-4272178
https://github.com/Sixze/ALS-Refactored/discussions/19#discussioncomment-1348386
